src/dune

```
@@ -28,8 +28,12 @@
  ; (promote (until-clean))
 
  (preprocess
+  ; The `pps` field is a list of ppx to apply to the sources before generating JavaScript. melange.ppx allows to use Melange attributes [@mel. ...] (https://melange.re/v2.0.0/communicate-with-javascript/#attributes)
+  ; reason-react-ppx allows to use JSX for ReasonReact components
```

With this line wording it might not be too clear what the connection between JSX and ppxs/attributes. Maybe st like this? Also, should we link to https://reasonml.github.io/docs/en/jsx?

```suggestion
  ; reason-react-ppx allows to define ReasonReact components by using the [@JSX] attributes,
  ; which are automatically generated by Reason syntax.
```

---

I would avoid mention [@JSX] while explaining what's pp

---

I re-thought about this, and added both with the reason link